agcaballero/walkthroughs/week-4-libraries/maps/examples/hello-world/src/main/webapp/index.html

```diff
@@ -2,7 +2,7 @@
 <html>
 <head>
   <title>Hello Google Maps</title>
-  <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyAEQd51hndr1lX__sO6RauEwa5Owi7AILY"></script>
+  <script src="https://maps.googleapis.com/maps/api/js?key=YOUR_API_KEY"></script>
   <script src="script.js"></script>
   <link rel="stylesheet" href="style.css">
```

Comment:
Yes, but it's showing up as having been removed from the commit for me, so it's strange you can still see it. I'll go check on it manually to make sure that it's not a part of the commit.

Reply:
It does seem to have been added, but since it doesn't directly affect my project I'm just going to leave it. Thank you for pointing it out though I'll be more careful next time!

Reply:
Done